Add landing pages (#1801)
spryker/spryker-docs: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/episerver/episerver.md

---
title: Episerver
description: This section lists documents inside the Episerver section.
last_updated: Apr 26, 2023
template: concept-topic-template
---

> The *Episerver* section includes the following documents:

Reviewer: Can you add a few words on what this section is about?

Author: ```suggestion
The "Episerver" section provides guidance on integrating and utilizing Episerver. Learn about the Episerver API requests and order reference commands. Follow the instructions to integrate and configure Episerver in your project.

For more information the following documents in this section:
```

---
title: Episerver
description: This section lists documents inside the Episerver section.
last_updated: Apr 26, 2023
template: concept-topic-template
---

> The "Episerver" section provides guidance about integrating and utilizing Episerver. Learn about the Episerver API requests and order reference commands. Follow the instructions to integrate and configure Episerver in your project.

Author: ```suggestion
The *Episerver* section provides guidance about integrating and utilizing Episerver. Learn about the Episerver API requests and order reference commands. Follow the instructions to integrate and configure Episerver in your project.
```

> The *Episerver* section provides guidance about integrating and utilizing Episerver. Learn about the Episerver API requests and order reference commands. Follow the instructions to integrate and configure Episerver in your project.

Reviewer: ```suggestion
The *Episerver* section provides guidance about integrating, configuring, and utilizing Episerver.
```
